### ISSUE TITLE
Not handling quoted yaml

### ISSUE THREAD

**Issue by vorpalhex** [mbround18/valheim-docker#14]
When you supply env values in docker-compose, it's valid to wrap strings in double quotes (and recommended because yaml sometimes casts values in funny ways). However the current templating for `start_server_rusty.sh` is not correctly handling this and instead producing double quotes:

```
# Launch Command
/home/steam/valheim/valheim_server.x86_64 \
    -port "2456" \
    -name ""myservername"" \
    -world ""Dedicated"" \
    -password ""somepasswordhere"" \
    -public 1 \
    2>&1 | tee ./output.log  > /dev/null 2>&1 &
    ```
    
 This does launch valheim but since password will be parsed as `""` the server will never be registered or appear.

**Comment by mbround18** [comment]
Can you provide an example of your docker-compose?

**Comment by vorpalhex** [comment]
```
version: "3.3"
services:
  valheim:
    container_name: valheim
    hostname: valheim
    image: mbround18/valheim:latest
    ports:
      - 2456:2456/udp
      - 2457:2457/udp
      - 2458:2458/udp
    environment:
      - PUID=997
      - PGID=997
      - TZ=America/Chicago
      - NAME="somenamehere"
      - WORLD="Dedicated"
      - PORT=2456
      - PASSWORD="somepasswordhere"
    restart: unless-stopped
    labels:
      - "traefik.enable=false"
    volumes:
      - /var/docker/valheim_saves:/home/steam/.config/unity3d/IronGate/Valheim
      - /var/docker/valheim_server:/home/steam/valheim
      ```

**Comment by mbround18** [comment]
I see, my environment variables are following a different spec. For mine they are listed as:
```yaml
environment:
  NAME: "Example"
  PASSWORD: "SomthingNordic"
```

**Comment by mbround18** [comment]
I can fix this after work, should be ~5-6pm PST

**Comment by vorpalhex** [comment]
I was able to just unquote my variables for now so no rush, but thanks for the prompt attention.

**Comment by mbround18** [comment]
@vorpalhex I just merged #17 which should handle this but I suspect you will run into #16 as well based off the compose example you provided.

**Comment by mbround18** [comment]
I am going to close this issue, I have tested this via: 
- Docker Compose locally
- Base Docker locally
- Odin via CLI on an ubuntu VPS
- Odin via Environment Variables on an ubuntu VPS
